docs/ReleaseNotes/Kubernetes-Operator-for-PostgreSQL-RN2.5.1.md

```diff
@@ -0,0 +1,29 @@
+# Percona Operator for PostgreSQL 2.5.1
+
+* **Date**
+
+    March 03, 2025
+
+* **Installation**
+
+    [Installing Percona Operator for PostgreSQL](../System-Requirements.md#installation-guidelines) 
+
+
+## Release highlights
+
+
+This release fixes the [CVE-2025-1094 :octicons-link-external-16:](https://www.postgresql.org/support/security/CVE-2025-1094/), vulnerability in the libpq PostgreSQL client library, which made images used by the Operator vulnerable to SQL injection within the PostgreSQL interactive terminal due to the lack of neutralizing quoting. For now, the fix includes the image of PostgreSQL 16.8  and other database cluster images based on PostgreSQL 16.8.
+
+## Supported platforms
+
+The Operator was developed and tested with PostgreSQL versions 12.20, 13.16, 14.13, 15.8, 16.4, and 16.8. Other options may also work but have not been tested. The Operator 2.5.1 provides connection pooling based on pgBouncer 1.23.1 and high-availability implementation based on Patroni 3.3.2.
```

Comment:
pgbouncer 1.24.0 + We agreed to mention that pmm client version is also updated, or this goes only to images list?